src/Plugin/Field/FieldFormatter/StrawberryMediaFormatter.php

```diff
@@ -215,41 +216,39 @@ public function viewElements(FieldItemListInterface $items, $langcode) {
           if (isset($mediaitem['type']) && $mediaitem['type'] == 'Image') {
             if (isset($mediaitem['dr:fid'])) {
               // @TODO check if loading the entity is really needed to check access.
-              $file = OcflHelper::resolvetoFIDtoURI(
-                $mediaitem['dr:fid']
-              );
+              $file = OcflHelper::resolvetoFIDtoURI($mediaitem['dr:fid']);
 
-              //@TODO if no media key to file loading was possible
-              // means we have a broken/missing media reference
-              // we should inform to logs and continue
+              //@TODO if no media key to file loading was possible means we have
+              // a broken/missing media reference we should inform to logs and
+              // continue.
               if (!$file) {
                 continue;
-
               }
               if ($this->checkAccess($file)) {
-                $iiifidentifier = urlencode(
-                  StreamWrapperManager::getTarget($file->getFileUri())
-                );
+                $iiifidentifier = urlencode(StreamWrapperManager::getTarget($file->getFileUri()));
                 if ($iiifidentifier == NULL || empty($iiifidentifier)) {
                   continue;
                 }
-                // ImageToolKit use the $file->getFileUri(), we don't want that yet
+                // ImageToolKit use the $file->getFileUri(), we don't want that
+                // yet.
                 // @see https://github.com/esmero/format_strawberry/issues/1
 
-                //@ TODO recheck cache tags here, since we are not really using the file itself.
-                $filecachetags = $file->getCacheTags();
```

Comment:
Wonder if we should add the file cache tags instead of removing this? (Given its and not Solved `@TODO`) or do you think it is not needed?

Reply:
Aw sorry about that, we're not doing anything with the tags so maybe some more work is needed for this?

Reply:
No problem. Let's keep the //@TODO for now and remove 240. We can work on that in another issue. Thanks

Reply:
Hey. I will merge this. We can deal with the @TODO another day. I have done the cache stuff on the WARC formatter so we can copy from there. Thanks!

Reply:
I've added the cache tags to the element delta, is that what would be intended?